content/teams/Utopians/index.md

```diff
@@ -0,0 +1,101 @@
+---
+type: "team"                   
+title: Utopians
+name: "Utopians"
+github-link: "https://github.com/Karthikeshwar1/CodeCamp1.0"
+description: "A project by utopians"
+Team-Quote: "BeLikeUtopians"
+Team_Tech_Stack: "Python, OpenCV"
+Application_Type: "Desktop application"
+No_Of_Member: "4"
+---
+
+Hey ! We are Team Utopians. We are here to win. and we also love open source.
+
+---
+
+## Project's Overview
+
+_**Theme-Name**_ : Gesture recognition
```

Comment:
Mention the Themes here .. i.e from our website. from which your application belong.